infra/cardano-sl-infra.cabal

```diff
@@ -29,9 +29,7 @@ library
                         Pos.Infra.Reporting.Exceptions
                         Pos.Infra.Reporting.Health.Types
                         Pos.Infra.Reporting.Http
-                        Pos.Infra.Reporting.MemState
                         Pos.Infra.Reporting.Methods
-                        Pos.Infra.Reporting.Metrics
```

Comment:
Moved to [here](https://github.com/input-output-hk/cardano-sl/pull/3254/files#diff-6a285943be035601fced61b34310af23R173)